### DISCUSSION TITLE
got a critical info including"argument invalid"

### DISCUSSION THREAD

**Discussion by 619954559** [Flexget/Flexget#3082] [Q&A]
i had run flexget for 2days, and got a critical error 

Unable to download the RSS for task pterfree (https://pterclub.com/torrentrss.php?rows=20&isize=1&linktype=dl&passkey=cac54a47aac4d7b4c47): ('Connection aborted.', OSError(22, 'Invalid argument'))

and

Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', OSError(22, 'Invalid argument'))': /viewpeerlist.php?id=111512

**Comment by paranoidi** [comment]
Fyi, your passkey is exposed

**Reply by 619954559** [reply]
thx,bro~ i've delete serveal letters XD
